TCAT/Cells/RouteTableViewCell.swift

```diff
@@ -31,121 +31,236 @@ class RouteTableViewCell: UITableViewCell {
     }
 
-    // MARK: Reuse
+    private func styleBottomBorder() {
+        bottomBorder.backgroundColor = .lineDotColor
+    }
 
-    override func prepareForReuse() {
-        routeDiagram.prepareForReuse()
+    private func styleCellSeparator() {
+        cellSeparator.backgroundColor = .tableBackgroundColor
+    }
+    
+    // MARK: Add subviews
+    
+    func addRouteDiagramSubviews() {
+        routeDiagram.addSubviews()
+    }
+
+    // MARK: Activate constraints
 
-        routeDiagram.removeFromSuperview()
-        cellSeperator.removeFromSuperview()
-        bottomBorder.removeFromSuperview()
+    func activateRouteDiagramConstraints() {
+        routeDiagram.activateConstraints()
+    }
+    
+    private func activateConstraints() {
+        setTranslatesAutoresizingMaskIntoConstraints()
+        setDebugIdentifiers()
         
-        hideLiveElements()
+        NSLayoutConstraint.activate([
+            topBorder.topAnchor.constraint(equalTo: contentView.topAnchor),
+            topBorder.leadingAnchor.constraint(equalTo: contentView.leadingAnchor),
+            topBorder.trailingAnchor.constraint(equalTo: contentView.trailingAnchor),
+            topBorder.heightAnchor.constraint(equalToConstant: cellBorderHeight),
+            topBorder.bottomAnchor.constraint(equalTo: verticalStackView.topAnchor)
+        ])
         
-        // stop timer
-        timer?.invalidate()
+        NSLayoutConstraint.activate([
+            verticalStackView.leadingAnchor.constraint(equalTo: contentView.leadingAnchor),
+            verticalStackView.trailingAnchor.constraint(equalTo: contentView.trailingAnchor),
+            verticalStackView.bottomAnchor.constraint(equalTo: bottomBorder.topAnchor)
+        ])
+        
+        NSLayoutConstraint.activate([
+            bottomBorder.leadingAnchor.constraint(equalTo: contentView.leadingAnchor),
+            bottomBorder.trailingAnchor.constraint(equalTo: contentView.trailingAnchor),
+            bottomBorder.heightAnchor.constraint(equalToConstant: cellBorderHeight),
+            bottomBorder.bottomAnchor.constraint(equalTo: cellSeparator.topAnchor)
+        ])
+        
+        NSLayoutConstraint.activate([
+            cellSeparator.leadingAnchor.constraint(equalTo: contentView.leadingAnchor),
+            cellSeparator.trailingAnchor.constraint(equalTo: contentView.trailingAnchor),
+            cellSeparator.heightAnchor.constraint(equalToConstant: cellSeparatorHeight),
+            cellSeparator.bottomAnchor.constraint(equalTo: contentView.bottomAnchor)
+        ])
+        
+        NSLayoutConstraint.activate([
+            arrowImageView.heightAnchor.constraint(equalToConstant: arrowImageViewHeight),
+            arrowImageView.widthAnchor.constraint(equalToConstant: arrowImageViewWidth),
+            arrowImageView.centerYAnchor.constraint(equalTo: departureTimeLabel.centerYAnchor)
+        ])
+    }
+    
+    private func setTranslatesAutoresizingMaskIntoConstraints() {
```

Comment:
Just a thought, what if you made an array with all of these UIViews, and did a for loop?

    let viewArray = [ 
        ... 
    ]

    viewArray.forEach { $0.translatesAutoresizingMaskIntoConstraints = false }

Reply:
True that is shorter